README.md

```diff
@@ -37,12 +37,14 @@ Since Pulsar 2.6.2, KoP version changes with Pulsar version accordingly. The ver
 
 | KoP version | Pulsar version |
 | :---------- | :------------- |
-| [2.8.0](https://github.com/streamnative/kop/releases/tag/v2.8.0) |Pulsar 2.8.0|
-| [2.7.0](https://github.com/streamnative/kop/releases/tag/v2.7.0) |Pulsar 2.7.0|
-| [2.6.2](https://github.com/streamnative/kop/releases/tag/v2.6.2) |Pulsar 2.6.2|
-| [0.3.0](https://github.com/streamnative/kop/releases/tag/v0.3.0) |Pulsar 2.6.1|
-| [0.2.0](https://github.com/streamnative/kop/releases/tag/v0.2.0) |Pulsar 2.5.0|
-| [0.1.0](https://github.com/streamnative/kop/releases/tag/v0.1.0) |Pulsar 2.5.0|
+| [2.8.1](https://github.com/streamnative/kop/releases/tag/v2.8.1.0) |Pulsar 2.8.1|
+| [2.8.0](https://github.com/streamnative/kop/releases/tag/v2.8.0.1) |Pulsar 2.8.0|
+
+**We highly recommend using KoP 2.8.0 or higher.** You can also find other releases from [here](https://github.com/streamnative/kop/releases), including the old versions.
```

Comment:
It's explained in `docs/upgrade.md`, the `For KoP < 2.8.0` section.

Reply:
Could we update it to : **It is highly recommend to use KoP 2.8.0 or higher because there is a breaking change since KoP 2.8.0. For details, see [upgrade.md](docs/upgrade.md).**

Reply:
Sure. This description is great!

Reply:
`It is highly recommend` -> `It is highly recommended`

Reply:
yes, it is.